corehq/apps/export/models/new.py

```diff
@@ -1209,6 +1212,10 @@ def type(self):
         return FORM_EXPORT
 
     @classmethod
+    def schema_version(cls):
+        return FORM_DATA_SCHEMA_VERSION
```

Comment:
i tried to do this, but it wouldn't place nicely with jsonobject. it kept wanting to wrap it